### PR TITLE
[DOCFIX] Add doc on how to deploy proxy in docker/k8s

### PR DESCRIPTION
**Pull request by ssz1997**
Solves https://github.com/Alluxio/alluxio/issues/13137

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 1]
```diff
@@ -1639,6 +1639,38 @@ spec:
 For more information on how to configure a pod to use a persistent volume for storage in Kubernetes,
 please refer to [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/).
 
+### Set up Alluxio Proxy server
```
Comment (HelloHorizon) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1642]:
```suggestion
### Start Alluxio Proxy server
```

Comment (HelloHorizon) [reply docs/en/deploy/Running-Alluxio-On-Kubernetes.md:1642]: Set up is hard to follow. It will be better to change to sub-steps, like change configurations, start the proxy server, stop proxy server, etc.

**Review by HelloHorizon — CHANGES_REQUESTED** [review]
@ssz1997 thank you for doc! Left some minor comments

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -498,6 +498,23 @@ See [Fuse configuration]({{ '/en/api/POSIX-API.html' | relativize_url }}#configu
 and [Fuse mount options]({{ '/en/api/POSIX-API.html' | relativize_url }}#configure-mount-point-options)
 for more details about how to modify the Fuse mount configuration.
 
+### Set up Alluxio Proxy
```
[the remark this answers is not in the document]
Comment (ssz1997) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:501]: I don't think we have a Proxy page...

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 3]
```diff
@@ -498,6 +498,23 @@ See [Fuse configuration]({{ '/en/api/POSIX-API.html' | relativize_url }}#configu
...
+    --net=host \
+    --name=alluxio-proxy \
+    --security-opt apparmor:unconfined \
+    -e ALLUXIO_JAVA_OPTS=" \
+       -Dalluxio.master.hostname=localhost" \
+    alluxio/{{site.ALLUXIO_DOCKER_IMAGE}} proxy
+```
+
+See [Properties List](https://docs.alluxio.io/os/user/edge/en/reference/Properties-List.html) for more
+configuration options for Alluxio proxy server.
```
[the remark this answers is not in the document]
Comment (ssz1997) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:516]: The properties for proxy are all under "Common Configuration" and doesn't have its own section. I think it makes for proxy to has its own section

**Review by HelloHorizon — APPROVED** [review]
LGTM

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.9 please
